pyvista/plotting/plotting.py

```diff
@@ -1387,6 +1387,16 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
             If set to ``True``, then the number of unique values in the scalar
             array will be used as the ``n_colors`` argument.
 
+        silhouette : dict, bool, optional
+           As a dict, it contains the properties of the silhouette to display:
+               * ``color`` (color of the silhouette)
+               * ``line_width`` (edge width)
+               * ``opacity`` (edge transparency between 0 and 1)
+               * ``feature_angle`` (bool to display sharp edges)
+               * ``decimate`` (level of decimation between 0 and 1 or None)
+           If True, the default values are used and if False, no silhouette
+           is displayed. Defaults to False.
```

Comment:
There's a space missing from the indentation of this block. More importantly, an empty line is missing before the list for correct rendering.
```suggestion
            As a dict, it contains the properties of the silhouette to display:

                * ``color`` (color of the silhouette)
                * ``line_width`` (edge width)
                * ``opacity`` (edge transparency between 0 and 1)
                * ``feature_angle`` (bool to display sharp edges)
                * ``decimate`` (level of decimation between 0 and 1 or None)
            If True, the default values are used and if False, no silhouette
            is displayed. Defaults to False.
```

The literals `False` and `None` don't have code markdown here, although that's pretty inconsistent all across the module anyway.

Reply:
It's not an excuse, I'll fix it :+1:

Reply:
Should be good now